web-backend/src/main/resources/application.yml

```diff
@@ -4,6 +4,12 @@ spring:
   docker:
     compose:
       lifecycle-management: start_only
+  datasource:
```

Comment:
We don't want to do this. Assuming we use the Docker Compose DB, the service connection should be auto injected. If we put it here, we've now hard code configured our app to only work with this connection, the setup steps for which are not documented anywhere.